tests/Microsoft.Identity.Web.Test/TokenAcquisitionAddInTests.cs

```diff
@@ -1,74 +1,62 @@
+using System;
 using Microsoft.Identity.Abstractions;
 using Microsoft.Identity.Client;
+using Microsoft.Identity.Web.Test.Common.Mocks;
+using Microsoft.Identity.Web.Test.Common;
 using NSubstitute;
 using Xunit;
+using System.Threading.Tasks;
 
 namespace Microsoft.Identity.Web.Tests
 {
     public class TokenAcquisitionAddInTests
     {
         [Fact]
-        public void InvokeOnBuildConfidentialClientApplication_InvokesEvent()
+        public async Task InvokeOnBeforeTokenAcquisitionForApp_InvokesEvent()
         {
             // Arrange
-            var options = new TokenAcquisitionAddInOptions();
+            var options = new TokenAcquisitionExtensionOptions();
             var acquireTokenOptions = new AcquireTokenOptions();
-            ConfidentialClientApplicationBuilder builderMock = null!;
+            acquireTokenOptions.ForceRefresh = true;
 
-            bool eventInvoked = false;
-            options.OnBuildConfidentialClientApplication += (builder, options) =>
+            //Configure mocks
+            using MockHttpClientFactory mockHttpClient = new();
+            using (mockHttpClient.AddMockHandler(MockHttpCreator.CreateClientCredentialTokenHandler()))
+            using (mockHttpClient.AddMockHandler(MockHttpCreator.CreateClientCredentialTokenHandler()))
             {
```

Comment:
Since this `using` encapsulates the rest of the test, you don't need brackets which will reduce a level of indentation.
```suggestion
```